templates/mongodb.conf.4.erb

```diff
@@ -0,0 +1,168 @@
+#mongodb.conf - generated from Puppet
```

Comment:
since this is a new template, can you please convert it to epp? We require that for all new templates.